### DISCUSSION TITLE
How can I make Lua script do something on tick?

### DISCUSSION THREAD

**Discussion by dnqbob** [o3de/o3de#6582] [Help Wanted] [answered]
How can I make Lua script do something on tick? I could not find any clue on this on docs.o3de.org

Now Lua Script only has 
OnActivate()
OnDeactivate()

I don't know if it is all Lua Script has, because script component in Unity/Unreal always has a Ticks/Update for action that needs to be executed per tick

**Comment by DavidHammertim** [comment, marked as answer]
[The wiki's ](https://github.com/o3de/o3de/wiki) "Hello World" goes over the bare minimum for a lua script and "TickBus" goes over delta time.
